Autopilot/TelemetryManager/telemetryManager.hpp

```diff
@@ -0,0 +1,17 @@
+#pragma once
+#include "telemetryStateManager.hpp"
+
+class telemetryState;
+
+class telemetryManager
+{
+    public:
+        telemetryManager();
+        inline telemetryState* getCurrentState() const{return currentState;}
+        void execute();
+        void setState(telemetryState& newState);
+        bool dataValid;
+        bool dataError;
+    private:
+        telemetryState* currentState;
+};
```

Comment:
At the end of C and C++ files there needs to be a newline character at the end of the file. This minus sign means the file is missing it. To fix just add a line or two after this closing bracket :))

Reply:
(It's not appearing here. but go to the "Files changed" section and you'll see it)

Reply:
Got it.